source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5002,6 +5013,17 @@ export class ActionSet extends CardElement {
         return this._actionCollection.getResourceInformation();
     }
 
+    /**
+     * @inheritdoc
+     */
+    findObject(node: Node): CardObject | undefined {
+        let target;
+        for(const action of this._actionCollection.items) {
+            target = target ?? action.findObject(node); // recur through Actions
+        }
+        return target ?? super.findObject(node); // else, check ActionSet
```

Comment:
The comment doesn't seem to make sense. super.findObject will run the ancestor class' logic, which isn't ActionSet.

Also:
- A more useful comment would be "If a target wasn't fond in one of the ActionSet's actions, defer to the parent implementation."
- Can we place comments above the line it refers to?